js/index.js

```diff
@@ -102,6 +113,38 @@ const searchAction = ({
   },
 }) => {
   //used destructuring
-  const search = value;
-  searchRequest(search);
+  search = value;
+  searchRequest();
+};
+
+const moreContent = () => {
+  if (pageContent === 'trending') trendingRequest(true);
+  else if (pageContent === 'search') searchRequest(search, true);
+};
+
+//Event listeners
+
+input.addEventListener('keyup', event => {
+  //used destructuring for event.keyCode => event object is destructured and {keyCode} takes it's value.
+  if (event.keyCode === 13) {
+    search = event.target.value;
+    searchRequest();
+  }
+});
+
+window.onclick = ({ target }) => {
+  //used destructuring for event.target => event object is destructured and {target} takes it's value.
+  if (target == modalContent) {
+    modalContent.innerHTML = '';
+    modal.style.display = 'none';
+  }
 };
+
+window.addEventListener('scroll', () => {
+  if (
+    window.scrollY + window.innerHeight + 1 >=
+    document.documentElement.scrollHeight
+  ) {
```

Comment:
Is that prettified? 🤔

Reply:
Yes, that is what Prettier did to this snippet. I actually wrote it one line.

Reply:
Looks weird but okay then..